custom_components/ha_washdata/frontend.py

```
@@ -221,3 +231,98 @@ async def _on_lovelace_loaded(event: Event) -> None:
+    # before HA fires EVENT_PANELS_UPDATED and the frontend tries to import it.
+    # Guard with a flag set *before* the await so two concurrent setup_entry
+    # calls (multiple devices) don't both try to register the same route and
+    # log a benign "method GET is already registered" debug line.
+    if not hass.data.get(PANEL_STATIC_REGISTERED):
+        hass.data[PANEL_STATIC_REGISTERED] = True
+        try:
+            from homeassistant.components.http import StaticPathConfig  # pylint: disable=import-outside-toplevel
+
+            if hasattr(hass.http, "async_register_static_paths"):
+                await hass.http.async_register_static_paths(
+                    [StaticPathConfig(PANEL_JS_URL, str(src), True)]
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)